File: Aula28022019_TesteUnidade/ConsoleApp1/ConsoleApp2/Triangulo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ConsoleApp2
{
    class Triangulo
    {
        public Triangulo()
        {

        }
        public String VerificarTipo(int l1, int l2, int l3)
        {
            if ((l1 == l2) && (l2 == l3) && (l1 == l3))
            {
                return ("Equilatero");
            }
            else
            {
                if ((l1 != l2) && (l2 != l3) && (l1 != l3))
                {
                    return ("Escaleno");
                }
                else
                {
                    return ("Isósceles");
                }
            }
        }
    }
}
}

File: Aula28022019_TesteUnidade/TesteUnidadeJUnit/src/javaapplication3/Triangulo.java
package javaapplication3;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author Aluno
 */
public class Triangulo {
    
    public Triangulo()
    {
        
    }
    
    // verificar dados
    // verificar a soma

    public String VerificarTipo(int l1, int l2, int l3)
    {
     if((l1==l2) &&(l2==l3)&&(l1==l3))
     {
         return("Equilatero");
     }
     else
     {
         if ((l1 != l2) && (l2 != l3) && (l1 != l3)) 
         {
             return ("Escaleno");
         } 
         else 
         {
             return ("Isósceles");
         }
     }
    }
}
